packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.1]
+
+### Changed
+
+- update the types described in `types/on-chain-notification/schema` and `types/on-chain-notification/on-chain-notification` ([#4818](https://github.com/MetaMask/core/pull/4818))
```

Comment:
Could you describe the nature of these changes? How do these types affect users of this package?

It's not relevant here which internal types are changed. We should be describing how these changes impact someone who uses this package. Are these types part of the public API of the package? Are these breaking changes?

Reply:
Agreed, these types/schema changes (non-breaking) are important. They introduce new notifications that we are adding onto extension and mobile.

Reply:
@matteoscurati can you update the changelog to represent these in the next release (tomorrow)

Reply:
@matteoscurati can you update the changelog to be more descriptive on this change?

Reply:
@Gudahtt @Prithpal-Sooriya thanks for the comment. You’re right. I’ve improved the description to better explain the non-breaking changes implemented. Can I ask you for a new review?